css/style.css

```diff
@@ -65,12 +129,142 @@
     text-transform: uppercase;
   }
 
+  .toggle img {
+    margin-top: 23px;
+  }
+
   #social-links img {
     width: 20px;
     height: 20px;
     padding: 10px 15px 1px 1px;
     cursor: pointer;
   }
+
+  .tools {
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    list-style-type: none;
+    padding: 5px 1px;
+    margin: 10px 1px;
+  }
+
+  .tools li {
+    padding: 4px 12px 4px 12px;
+    border-radius: 8%;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    background-color: #ebebff;
+    color: #6070ff;
+    margin-right: 15px;
+  }
+
+  .see-project-btn {
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    color: #396df2;
+    border: solid #6070ff 1px;
+    border-radius: 8px;
+    padding: 12px;
+    width: 126px;
+    height: 48px;
+    margin: 10px 1px;
+    background: white;
+  }
```

Comment:
- [ ] Kindly make sure the `button` and link `a` tags follow the button styling: Active, Hover and Pressed(see image) to stick to the Figma design.
![Screenshot from 2023-03-24 18-26-25](https://github.com/shimels1/portfolio_microverse/assets/91142432/0f48d1d8-81a1-4ace-bed1-f6914bd2f232)